gems/aws-sdk-cloudfront/spec/signer_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
```

Comment:
The public interfaces are cookie signer and url signer, not the signer module. Signer is an included module to share implementation. I don't think this file should exist, and I think cookie signer and url signer need to test both key types (RSA / ECDSA) and argument presence (private key, private key path, key pair id). This pattern is better because url signer and cookie signer (or other signer) implementations can be refactored without modifying tests.

Reply:
When I was reviewing the existing specs, `CookieSigner` did not contain any tests relating to validating RSA key/arguments presence and only `UrlSigner` had them so I wanted to consolidate the shared behavior within `signer_spec` - that being said, I'm fine with just adding these into both existing spec files